src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -42,6 +42,34 @@ define(function (require, exports, module) {
     var STEP_MULTIPLIER = 5;
 
     /**
+     * Convert 0x notation into hex6 format for tinycolor
+     * compatibility: ("0xFFAACC" => "#FFFFFF")
+     */
+    function color0xToHex(color,toStr){
```

Comment:
Not a huge fan of the second parameter, maybe rename it to something like `convertToString` and make it a boolean 🤔